lms/templates/courseware/course_footer.html

```diff
@@ -0,0 +1,7 @@
+<div class="container course-footer">
+  % if settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False) and course.licensable and course.license:
+    <div class="xmodule_licensable">
+      ${course.license.html}
```

Comment:
HTML should be written into this template, not generated in the `license.html` attribute.